product-mini/platforms/linux-sgx/enclave-sample/App/README.md

```diff
@@ -2,90 +2,48 @@
 
 ## Build WAMR vmcore (iwasm) for Linux-SGX
 
-### SIM Mode
+Please follow [this guide](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/doc/linux_sgx.md#build-wamr-vmcore-iwasm-for-linux-sgx) to build iwasm as the prerequisite.
 
-The default SGX mode in WAMR is the SIM mode. Build the source code and enclave example, please refer to [this guild](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/doc/linux_sgx.md#build-wamr-vmcore-iwasm-for-linux-sgx).

```

Comment:
want to confirm we do want to remove this paragraph?

Reply:
I'll give the explanation about this in new PR.